docs/reference/ddl/future.rst

```diff
@@ -0,0 +1,81 @@
+.. _ref_eql_ddl_future:
+
+===============
+Future Behavior
+===============
+
+This section describes the DDL commands pertaining to
+:ref:`future <ref_datamodel_future>`.
+
+
+Create future
+=============
+
+:eql-statement:
+
+Enable a particular future behavior for the current schema.
+
+.. eql:synopsis::
+
+    create future <FutureBehavior> ";"
+
+There's a :ref:`corresponding SDL declaration <ref_eql_sdl_future>`
+for enabling a future behavior, which is the recommended way of doing this.
+
+Description
+-----------
+
+The command ``create future`` enables the specified future behavior for
+the current database.
+
+Examples
+--------
+
+Enable simpler non-recursive access policy behavior :ref:`non-recursive access
+policy <ref_datamodel_access_policies_nonrecursive>` for the current schema:
+
+.. code-block:: edgeql
+
+    create future nonrecursive_access_policies;
+
+
+drop future
+===========
+
+:eql-statement:
+
+
+Stop importing future behavior prior to the EdgeDB version in which it appears.
+
+.. eql:synopsis::
+
+    drop future <FutureBehavior> ";"
+
+
+Description
+-----------
+
+The command ``drop future`` disables a currently active future behavior for
+the current database. However, this is only possible for versions of EdgeDB
+when the behavior in question is not officially introduced. Once a particular
+behavior is introduced as the standard behavior in an EdgeDB release, it
+cannot be disabled. Running this command will simply denote that no special
```

Comment:
> Running this command will simply denote that no special action is needed to enable it in this case.

I have a little trouble following this. Are you saying here that running `drop future` with a behavior that is now standard in your current release will have no effect but will tell you that the feature does not need to be enabled? If so, I'd propose:

> Running ``drop future`` in this scenario will have no impact on the database but will inform you that \<whatever it informs>

Also, I read your sentence to say that, when I try to *dis*able the feature, I'm going to get back a message telling me that I don't need to do anything to *en*able it. Is that accurate?